marathon.json

```diff
@@ -0,0 +1,13 @@
+{
+  "id": "hdfs-mesos",
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "image": "mesosphere/hdfs-mesos",
```

Comment:
We should come up with a scheme to use for tagging the docker images. Using `latest` will lead to issues down the road.